pkgs/development/python-modules/telepath/default.nix

```diff
@@ -1,20 +1,18 @@
 { buildPythonPackage
 , django
-, fetchFromGitHub
+, fetchPypi
 , lib
 , python
 }:
 
 buildPythonPackage rec {
   pname = "telepath";
-  version = "0.3";
+  version = "0.3.1";
   format = "setuptools";
 
-  src = fetchFromGitHub {
-    repo = "telepath";
-    owner = "wagtail";
-    rev = "v${version}";
-    sha256 = "sha256-kfEAYCXbK0HTf1Gut/APkpw2krMa6C6mU/dJ0dsqzS0=";
+  src = fetchPypi {
```

Comment:
changing to fetchPypi is fine
but let's reiterate

the reason why fetching from github is preffered is because usually published packages don't contain all files, eg tests

if upstream fails to tag a release, maintainer can always just use raw rev

hopefully this can help you make the best choice on how to proceed